ts-sdk/core/README.md

```diff
@@ -1 +1,71 @@
-# Orca Whirlpools Typescript Core
+# Orca Whirlpools Core SDK (WebAssembly)
+This package provides developers with advanced functionalities for interacting with the Whirlpool Program on Solana. Originally written in Rust, it has been compiled to WebAssembly (Wasm). This compilation makes the SDK accessible in JavaScript/TypeScript environments, offering developers the same core features and calculations for their Solana projects. The SDK exposes convenient methods for math calculations, quotes, and other utilities, enabling seamless integration within web-based projects.
+
+## Key Features
+- **WebAssembly Integration**: The original Rust-based SDK is compiled to Wasm, allowing it to be used in JavaScript and TypeScript projects for easy integration with web-based environments.
+- **Math Library**: Contains a variety of functions for math operations related to bundles, positions, prices, ticks, and tokens, including calculations such as determining position status or price conversions.
```

Comment:
Think we can remove `WebAssembly Integration` here since the Math library and the Quoting library are the webassembly integration.

I don't think it is important for a consumer to know that under-the-hood we use wasm though so we might consider making it less prominent (but not remove completely since you need to be using a version of node/web/bundler that supports wasm)